### DISCUSSION TITLE
Connecting to sql server (mssql)

### DISCUSSION THREAD

**Discussion by bulugani0613** [FlowiseAI/Flowise#1652] [Q&A]
Dear Team, 

In the front end I do see an option for mssql, however unable to connect to mssql server. 

Please let me know if there is support for mssql and also the parameters to pass for mssql when the connection is using integrated security. 

Thanks in advance. 

Thanks & Regards,
Kumar M

**Comment by itetrant** [comment]
I got the issue: DriverPackageNotInstalledError: SQL Server package has not been found installed. Try to install it: npm install mssql --save
I tried:npm install mssql --save
Then got error: npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:^

**Reply by LucasMatuszewski** [reply]
@itetrant  - Probably you have solved it already, but just for future/others:

Flowise uses `pnpm` so you may see this `workspace` error when using `npm` (I forgot to use pnpm today and also got this error)

If you cloned the repo (instead of installing the app with `npm install -g flowise` as in docs) you probably should get into your flowise folder, remove `node_modules` (just in case, `rm -rf node_modules`) and run `pnpm install` (you may need to install pnpm first: `npm i -g pnpm`), as it is in Readme.md

**Reply by aymenmed11** [reply]
hello i am running flowise in my localhost, i did install mssql package but still getting this error "ConnectionError: Failed to connect to mssql.us.umbhost.net:1433 - self-signed certificate" although with code it work, any guidance?

**Reply by jjamgmv** [reply]
I have the same issue but I cannot find documentation

**Reply by ghost** [reply]
This issue persists. Suggest someone from Flowise please provide a solution or a documentation.

**Reply by mcGeorge66** [reply]
Same issue, this is really bad. Any solutions or workarounds on that? It is a blocker in my project and I have to find a solution now.
